src/user/User.py

```diff
@@ -12,8 +12,25 @@ class User(object):
   name = None
   state = UserState.Nameless
   conversant = None
+  timestamp = None
   """docstring for User"""
   def __init__(self):
     super(User, self).__init__()
+
+  def setState(self, newState):
```

Comment:
this method can be simplified to:
self.state = newState

if newState is the enum and not the integer